resources/grafana-cfg.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: grafana-cfg
-  namespace: kube-system
-data:
-  grafana.ini: |
-    [server]
-    root_url = %(protocol)s://%(domain)s/web/grafana
-    [paths]
-    provisioning = /etc/grafana/provisioning
```

Comment:
if you remove this option, the ConfigMap based provisioning will not work.

Reply:
@eldios I've not removed anything, I just moved that configmap to grafana.yaml file, which now contains all grafana-related resources.

Reply:
I put all that configuration inside a ConfigMap cause otherwise it won't stick between PODs deletion. 
We should probably just put all the content you have in that file in the ConfigMap, I guess.

Reply:
I meant, there is still the same configmap, but definition moved from grafana-cfg.yaml to grafana.yaml.